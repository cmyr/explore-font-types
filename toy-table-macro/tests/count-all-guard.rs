use raw_types::Uint16;

toy_table_macro::tables! {
    CountAll<'a> {
        item_count: Uint16,
        #[count_all]
        items: [Uint16],
        other: Uint16,
    }
}

fn main() {}
