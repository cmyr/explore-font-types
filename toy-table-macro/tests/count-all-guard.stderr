error: #[count_all] annotation only valid on last field (TODO: validate before here)
  --> tests/count-all-guard.rs:3:1
   |
3  | / toy_table_macro::tables! {
4  | |     CountAll<'a> {
5  | |         item_count: Uint16,
6  | |         #[count_all]
...  |
9  | |     }
10 | | }
   | |_^
   |
   = note: this error originates in the macro `toy_table_macro::tables` (in Nightly builds, run with -Z macro-backtrace for more info)
